### ISSUE TITLE
Write tests verifying that c302 data matches data from PyOpenWorm

### ISSUE THREAD

**Issue by travs** [openworm/CElegansNeuroML#28]
These tests should use the NeuroML2 python library to read c302 and compare key features to the contents drawn from PyOpenWorm to be sure they are consistent.

Depends on #27 being completed.

**Comment by travs** [comment]
@aribrich are you able to make it to the hackathon this Sunday? If so, this would be a good issue to discuss relative to your work on #23.

**Comment by aribrich** [comment]
@travs I'm planning on going right now. I had submitted a program that would make this comparison, but I'm not sure where we stand on that issue at the moment. Because of that I haven't really done any coding for this in a while. What were you thinking in this regard?

**Comment by pgleeson** [comment]
The [issue I just created](https://github.com/openworm/PyOpenWorm/issues/152) is quite relevant for this issue & the pull request.

**Comment by travs** [comment]
@aribrich The issue @pgleeson is referring to will be resolved soon, and then we can move forward on getting your code incorporated. Is the program you're speaking of #23 ?

**Comment by travs** [comment]
@ahrasheed 
Your work on #27 is relevant to this as well. Would you be interested in taking this issue on, in tandem with @aribrich ?

**Comment by ahrasheed** [comment]
@travs Sure. The [comparison script](https://github.com/openworm/CElegansNeuroML/pull/23) by  @aribrich raises important questions over the differences between the two spreadsheets. 

I think this issue is redundant, considering that right now the difference between connectome in c302 and PyOpenWorm is the same difference that exists between the spreadsheets CElegansNeuronTables.xls and NeuronConnect.xls (source for PyOpenWorm). Once the data is pulled from PyOpenWorm into c302 (by using [this](https://github.com/openworm/CElegansNeuroML/pull/46) script)  the tests will essentially be checking if c302 worked correctly to generate correct NeuroML2 files, **not** making a comparison. And such tests already exist [here](https://github.com/openworm/CElegansNeuroML/tree/master/CElegans/pythonScripts/c302/unittests), except right now they are using CElegansNeuronTables.xls to read the data and not PyOpenWorm. So changing the data source in these tests will make them into the tests required in this issue. Constraints regarding changing the data source just yet are also addressed [here](https://github.com/openworm/CElegansNeuroML/pull/46).

Would like to know if you guys think otherwise.

**Comment by travs** [comment]
@ahrasheed That makes sense; perhaps we can rename the issue to reflect what this change would entail (something like "Change data source to PyOpenWorm in unit tests").

Since this is a separate issue form #27, I propose we keep this one open until the requisite changes are made in the unit tests (even if it is only a few lines).

So essentially, this also depends on https://github.com/openworm/PyOpenWorm/issues/162 and https://github.com/openworm/PyOpenWorm/issues/163, as you say.

Let me know if you think the renaming is correct/sufficient.

**Comment by ahrasheed** [comment]
Agree with the name change. Thanks

**Comment by pgleeson** [comment]
@ahrasheed @travs I agree with that assessment too...
